doc/lua_api.txt

```diff
@@ -845,6 +845,131 @@ from the available ones of the following files:
+* `loop=false`,                 --default
+* `gain=1.0`,                   --default. gets multiplied with
+                                --  the SimpleSoundSpec's 'gain' value
+* `pitch=1.0`,                  --default. the SimpleSoundSpec's
+                                --  'pitch' value is ignored, 
+                                --  so you have use this one instead.
+* `fade=0.0`                    --default. the SimpleSoundSpec's
+                                --  'fade' value is ignored, 
+                                --  so you have use this one instead.
+}
+
+
+Regarding conflicting parameters between the sound parameter table and the SimpleSoundSpec:
+
+NOTE!: While passing a `pitch` and `fade` value in Sound Parameters is optional, the 
+SimpleSoundSpec's `pitch` and `fade` are ignored by minetest.sound_play(), so if said 
+values are not the default '1','0' then it is required to pass the same values 
+again in the Sound Parameters table, to have identical results to using the 
+SimpleSoundSpec in a node definition.
+
+
+Regarding conflicting parameters within the sound parameter table:
+
+Several of the parameters conflict when provided simultaneously. In this case,
+arguments of higher priority result in lower priority arguments being ignored 
+if they conflict. The table above is ordered in descending priority of arguments 
+overriding each other:
+to_player,exclude_player,pos,object,max_hear_distance.
+
+to_player overrides all other arguments.
+exclude_player overrides all other arguments for the given player,
+unless they are also specified as to_player.
+
+If no to_player parameter was passed:
+
+pos overrides any given object. If no pos is given and an object is given, the object's 
+position will be retrieved and used instead as the pos argument. If a pos value is supplied 
+but invalid, the sound will not be played. If a valid pos has been found, the sound will be 
+played at it, audible to all players within the given max_hear_distance. If no max_hear_distance 
+is supplied, the default value of '32' is used.
+
+If no to_player,pos,object was passed:
+the sound will be played 'globally', i.e. locationless and to all players.
+NOTE!: If a function that is supposed to return a position failed and only returned nil to
+a value which is used to call minetest.play_sound with this `pos`, this does not count as 
+'invalid position of positional sound'; it acts as if no pos was specified at all, leading to 
+the sound being unintentionally played to all players.
+
+max_hear_distance only sets a 'true/false' cutoff regarding hearing the sound:
+It has no effect on the distance-based lowering of volume of the sound.
+
+The `loop` parameter has several quirks: It does not override `max_hear_distance`, which 
+decides whether the initial sound is heard. However, players who heard the initial sound 
+will continue to hear the looped sound at a low volume according to their distance from it,
```

Comment:
Couldn't this entire section be abbreviated to "even if the distance changes, this does not affect the volume of the sound"?